### PR TITLE
add python packages for geodata science: geopandas, rasterio, etc...

### PR DESCRIPTION
**Pull request by t0b3**
pytest passes
ebuilds work fine on my system

### CODE REVIEW

**Comment by AndrewAmmerlaan** [comment]
I'm getting a strange error compiling `Fiona`:
```
>>> Compiling source in /var/tmp/portage/dev-python/fiona-1.8.20/work/Fiona-1.8.20 ...
* python3_8: running distutils-r1_run_phase distutils-r1_python_compile
python3.8 setup.py build -j 12
Failed to get options via gdal-config: [Errno 2] No such file or directory: 'gdal-config'
A GDAL API version must be specified. Provide a path to gdal-config using a GDAL_CONFIG environment variable or use a GDAL_VERSION environment variable.
```

**Comment by AndrewAmmerlaan** [comment]
> I'm getting a strange error compiling `Fiona`:
> 
> ```
> >>> Compiling source in /var/tmp/portage/dev-python/fiona-1.8.20/work/Fiona-1.8.20 ...
> * python3_8: running distutils-r1_run_phase distutils-r1_python_compile
> python3.8 setup.py build -j 12
> Failed to get options via gdal-config: [Errno 2] No such file or directory: 'gdal-config'
> A GDAL API version must be specified. Provide a path to gdal-config using a GDAL_CONFIG environment variable or use a GDAL_VERSION environment variable.
> ```

Installing sci-libs/gdal fixed this. But now there are two more failures:
[fiona-1.8.20:20220117-094231.log](https://github.com/gentoo/sci/files/7880791/fiona-1.8.20.20220117-094231.log)
[rasterio-1.2.10:20220117-094233.log](https://github.com/gentoo/sci/files/7880793/rasterio-1.2.10.20220117-094233.log)

File: dev-python/affine/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>sci@gentoo.org</email>
+    <name>Gentoo Science Project</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
+    <name>Python</name>
```
Comment (AndrewAmmerlaan) [inline dev-python/affine/metadata.xml:10]: Can we drop the Python Project van the metadata files please. Packages in ::science are solely the responsibility of the Science Project (and sometimes external maintainers).

File: dev-python/snuggs/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <email>sci@gentoo.org</email>
+    <name>Gentoo Science Project</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
+    <name>Python</name>
+  </maintainer>
+  <longdescription lang="en">
+  Snuggs are s-expressions for Numpy
+</longdescription>
```
Comment (AndrewAmmerlaan) [inline dev-python/snuggs/metadata.xml:14]: Pkgcheck says:
```
  dev-python/snuggs
    RedundantLongDescription: metadata.xml longdescription closely matches DESCRIPTION
```

File: dev-python/affine/metadata.xml (hunk 1 above)
Comment (t0b3) [reply dev-python/affine/metadata.xml:10]: no problem

File: dev-python/snuggs/metadata.xml (hunk 2 above)
Comment (t0b3) [reply dev-python/snuggs/metadata.xml:14]: this was intentionally, but now I found a better description from debian.

**Comment by t0b3** [comment]
> Installing sci-libs/gdal fixed this. But now there are two more failures: [fiona-1.8.20:20220117-094231.log](https://github.com/gentoo/sci/files/7880791/fiona-1.8.20.20220117-094231.log) [rasterio-1.2.10:20220117-094233.log](https://github.com/gentoo/sci/files/7880793/rasterio-1.2.10.20220117-094233.log)

Thank you for pointing out this error. I detected the relevant gdal USE deps and added them. Let me know if tests pass now...

**Comment by AndrewAmmerlaan** [comment]
> > Installing sci-libs/gdal fixed this. But now there are two more failures: [fiona-1.8.20:20220117-094231.log](https://github.com/gentoo/sci/files/7880791/fiona-1.8.20.20220117-094231.log) [rasterio-1.2.10:20220117-094233.log](https://github.com/gentoo/sci/files/7880793/rasterio-1.2.10.20220117-094233.log)
> 
> Thank you for pointing out this error. I detected the relevant gdal USE deps and added them. Let me know if tests pass now...

Thanks :+1: 

I am still getting these small test failures, apart from that it is fine! I suspect the cause is a dependency that is either too old or too new, are you running `arch` or `~arch~?

**Comment by t0b3** [comment]
ok, feel free to give feedback on those test failures or disable those failing tests yourself :-)
thanks for merging in sci repo - do you feel any chance in getting this to main repo eventually?

**Comment by AndrewAmmerlaan** [comment]
> do you feel any chance in getting this to main repo eventually?

The ebuilds are excellent so I don't see a reason why not. We can keep them in ::sci for a bit, and maybe in the future move them up to ::gentoo.

**Comment by t0b3** [comment]
That sounds great to me. Let me know if there is anything I can improve.

**Pull request closed without merging**